docs/software.md

```diff
@@ -1,5 +1,4 @@
 ---
-title: DietPi Software Options
```

Comment:
Here we need to leave it, since the menu entry is "Overview", hence unspecific.

Reply:
Why this? 
The menu entry left side is "Software options" AND the following H1 header is "DietPi Software options".

Reply:
No the title is "Overview" in the left side navigation. Easiest is to check the live docs 😉: https://dietpi.com/docs/software/

Reply:
Aah right. I was in the wrong main tab ("Home").